app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::PostsController < ApplicationController
+  skip_before_action :authenticate_user!
```

Comment:
- You have done well so far, but please do not skip authentication for your controller actions, as we only want registered users to have access to the app. Please fix this 🙏 
**OPTIONAL** - To help with authentication for your rails api, I suggest you use the devise_jwt for authentication, you can follow this [guide](https://enlear.academy/how-to-create-a-rails-6-api-with-devise-jwt-46fa35085e85) on how to install devise_jwt in your project, please fix this 😄